pkgs/development/python-modules/pytest-doctestplus/default.nix

```diff
@@ -17,7 +18,10 @@ buildPythonPackage rec {
     sha256 = "6fe747418461d7b202824a3486ba8f4fa17a9bd0b1eddc743ba1d6d87f03391a";
   };
 
-  buildInputs = [ pytest ];
+  nativeBuildInputs = [
+    pytest
```

Comment:
pytest goes into buildInputs.

Also this package could be switched to pytestCheckHook.